javascript/index.js

```diff
@@ -35,7 +35,7 @@ const portfolioCards = [
     description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
     languages: ['html', 'css', 'JavaScript'],
```

Comment:
- [ ] It would be better to use a button tag or an `a` tag in the LETS connect since it sends the user to the contact form 
![image](https://user-images.githubusercontent.com/74744787/219767562-3c68300a-1827-435e-b95d-7d533a0c85b0.png)

Reply:
Hi @MeqdamAlqudah 

Please have  look at it now,
I have done the changes you suggested